### PR TITLE
KRACOEUS-8835: handling delete line from the wizard

### PR DESCRIPTION
**Pull request by leoherbie**

### CODE REVIEW

File: coeus-impl/src/main/resources/org/kuali/coeus/common/notification/impl/SendNotificationDialog.xml [hunk 1]
```diff
@@ -218,7 +218,7 @@
                         <bean parent="Kc-SendNotification-WizardButton-Back" p:actionLabel="Add More Recipients"
                               p:actionParameters="Kc-SendNotification-Wizard.step:0" p:order="20" />
                         <bean parent="Uif-SecondaryActionButton" p:actionLabel="Cancel" p:libraryCssClasses="btn,btn-link" p:methodToCall="cancelNotifications"
-                            p:dialogDismissOption="REQUEST" p:dialogResponse="false" p:order="30"/>
+                            p:dialogDismissOption="REQUEST" p:dialogResponse="false" p:order="30" p:successCallback="dismissDialog('Kc-SendNotification-Wizard');"/>
```
Comment (jwillia) [inline coeus-impl/src/main/resources/org/kuali/coeus/common/notification/impl/SendNotificationDialog.xml]: Do you know why the dialogDismissOption doesn't handle closing the dialog?

**Comment by leoherbie** [comment]
No idea.  I can change it to IMMEDIATE but then it doesn't hit the server.  I tried to call KcControllerService().dismissDialog() on the server but that causes things to blow up.  The only way I found to get the things they way they should be is the way I did it.  With out this javascript call the dialog disappears but the screen is not clickable and stays greyed out.

**Comment by jwillia** [comment]
looks good to me then

**Comment by gathreya** [comment]
this seems like a recent bug with dialogs to me. I am seeing this in other places as well.

**Pull request merged**